client/pages/OriginalDataPoint/OriginalDataPoint.tsx

```diff
@@ -28,13 +29,15 @@ const OriginalDataPoint = () => {
         cycleName,
       })
     )
-    return () => dispatch(OriginalDataPointActions.reset())
+    dispatch(OriginalDataPointActions.reset())
```

Comment:
why you removed unmount action

Reply:
my bad. there was a return type error and I let visual studio fix the issue. I've just pushed the fix.